src/libs/EmojiUtils.js

```diff
@@ -245,6 +245,15 @@ function suggestEmojis(text, limit = 5) {
     return [];
 }
 
+/**
+ * Validates that this message contains emojis
+ * @param {String} text
+ * @returns {Array}
+ */
+function isEmoji(text) {
```

Comment:
`contains` might be a better name for this

```suggestion
function containsEmoji(text) {
```

Reply:
Yea I would make this changes